projects/super_res/data/vsrdata.py

```diff
@@ -22,6 +22,10 @@ def __init__(self, channels, mode, length):
         c384_np = np.stack([c384[channel].values for channel in channels], axis = 2)
         c48_np = np.stack([c48[channel].values for channel in channels], axis = 2)
 
+        if logscale:
+            c384_np = np.log(c384_np - c384_np.min() + 1e-14)
```

Comment:
Do you know what floating point precision we are using? It's probably better to make this more robust via finfo, e.g. if we are using `torch.float32`

```
 torch.finfo(torch.float32).resolution
```
This returns 1e-06.

Reply:
Ok got it. Then you can do `np.finfo(arr.dtype).resolution`. What do you think?